core/platform.js

```diff
@@ -117,22 +144,29 @@ class Platform extends EventEmitter {
         }
     }
 
-    start () {
+    start(integrations) {
         if (this.statusFlag !== global.StatusFlag.NotStarted) {
             throw new Error($$`StartError`);
         }
 
         console.title(figlet.textSync(this.packageInfo.name.toProperCase()));
-
-        console.title(' ' + this.packageInfo.version);
+        console.title(` ${this.packageInfo.version}`);
         console.info('------------------------------------');
         console.warn($$`StartingSystem`);
 
         // Load system config
         console.warn($$`LoadingSystemConfig`);
+        this.modulesLoader.loadModule({
```

Comment:
Assuming this will be extended to allow for other configuration services in the future